test/torch/distribution/test_negative_binomial.py

```diff
@@ -50,7 +50,7 @@ def test_custom_neg_bin_cdf(total_count, probs, value):
 @pytest.mark.parametrize("probs", [0.1, 0.5, 0.8])
 @pytest.mark.parametrize("total_count", [3, 7, 100])
 @pytest.mark.parametrize("value", [0.1, 0.5, 0.9])
-def test_custom_neg_bin_icdf(total_count, probs, value):
+def test_custom_studentt_icdf(total_count, probs, value):
```

Comment:
why studentt? (the function body refers to NegBin still.

Reply:
ha! rebasing issue I suppose